src/settings/SettingsSelfExclusion.js

```diff
@@ -74,7 +74,12 @@ export default class SettingsSelfExclusion extends PureComponent {
 				success: true,
 				serverError: false,
 			});
-			setTimeout(() => this.setState({ success: false }), 3000);
+			setTimeout(() => {
```

Comment:
@borisyankov i didn't add . The setTimeout there is what you use for splash message in self exclusion. It was added by you . 😉 . Btw have refactored the var names.